docs/user_manual/boxes.md

```diff
@@ -0,0 +1,328 @@
+appear brighter than that of an unselected position.
+
+If a single position is selected, information about its contents and available actions is displayed near the
+Contents diagram. If multiple positions are selected, the information displayed changes to be a series of
+search boxes for quickly adding items to those positions.
+
+To select a single position, click on a circle in the Contents diagram. The circle will become brighter to
+indicate that it is selected. To unselect a single position, press Ctrl (Windows, Linus) or Command (Mac)
+while you click a selected circle.
+
+To select multiple positions, press Ctrl (Windows, Linux) or Command (Mac) while you click on the target
+circles in the Contents diagram. To unselect multiple positions, press Ctrl (Windows, Linux) or Command (Mac)
+while you click on the circles you wish to unselect. The circles will become less bright to indicate that they
+are not selected.
+
+To select an entire row or column, click on the row or column header. To select multiple rows or columns,
+press Ctrl (Windows, Linux) or Command (Mac) while you click on the target row or column headers.
+
+To select all odd columns, click the "Select Odd Columns" button below the Contents diagram.
+To select all even columns, click the "Select Even Columns" button below the Contents diagram.
+
+To unselect all positions, click the "List all Box Contents" button at the top right of the contents table,
+near the Search field.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% assign figure = figure | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Adding a Single Item to a Box" %}
+
+Select the target position in the Contents diagram. In the information section that appears adjacent to the
+Contents diagram (see figure {{figure}}), enter the item's name, alias, or barcode into the "Search" field and
+click "Lookup". 
+
+{% include userman-figure.md num=figure cap="Additional information when one position is selected"
+img="one-position-selected.png" %}
+
+Once the search completes, any matching items found will be in the "Results" dropdown. Select the target item
+from the dropdown and click "Update Position". After the update completes, the information about the item in
+the selected position will update with the target item's information.
+
+If the target position already contains an item, saving the new item to that position will cause the existing
+item's location to become unknown. If the existing item's location should be tracked, it should be moved to a
+new box before or after saving the new item to the target position.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% assign figure = figure | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Adding Multiple Items to a Box" %}
+
+To add multiple items to a box, go to the Edit Box page by clicking on the box name in the Boxes list page.
```

Comment:
Don't need to describe how to get to the Edit Box page here since it should be described in "Editing Boxes." Same in other sections below